packages/playground-examples/copy/es/TypeScript/Type Primitives/Nullable Types.ts

```diff
@@ -0,0 +1,95 @@
+//// { title: "Tipos Anulables", order: 3, compiler: { strictNullChecks: false } }
+
+// JavaScript tiene dos formas de declarar valores que no
+// existen, y TypeScript agrega sintaxis adicional lo que
+// permite aun más formas de declarar algo como opcional o
+// anulable.
+
+// Primero, la diferencia entre ambas primitivas de
+// JavaScript: undefined y null.
+
+// La primitiva indefinida `undefined` es cuando algo no
+// puede ser encontrado o modificado.
+
+const emptyObj = {}
+const anUndefinedProperty: undefined = emptyObj['anything']
+
+// La primitiva nula `null` está destinada a ser usada
+// cuando hay una falta consciente de un valor.
+
+const searchResults = {
+  video: { name: 'LEGO Movie' },
+  text: null,
+  audio: { name: 'LEGO Movie Soundtrack' },
+}
+
+// ¿Por qué no usar undefined? Principalmente, porque ahora
+// puede verificar que la propiedad text se haya incluido
```

Comment:
```suggestion
// puedes verificar que la propiedad text se haya incluido
```

making this consistent with the rest of docs